libgalois/include/katana/PropertyIndex.h

```diff
@@ -30,13 +30,14 @@ class KATANA_EXPORT PropertyIndex {
   // The set key type is either a node/edge id (all the keys in the actual set)
   // or a value representing the search key.
   using set_key_type = typename std::variant<
-      IndexID, bool, int64_t, double_t, std::string_view*>;
+      IndexID, bool, uint8_t, int64_t, double_t, std::string_view*>;
 
   // PropertyIndex::iterator returns a sequence of node or edge ids.
   using base_iterator = typename std::multiset<set_key_type>::iterator;
   class iterator : public base_iterator {
   public:
     explicit iterator(const base_iterator& base) : base_iterator(base) {}
+    iterator() : base_iterator(0) {}
```

Comment:
We're making assumptions about constructor for `multiset::iterator`, which is not part of public STL API and can change in the future. This is risky.

Reply:
There is a `do_all` path which requires this constructor. I intend to spend more time on figuring out the right way, but agree that this is risky.